charts/jaeger/templates/_helpers.tpl

```diff
@@ -419,3 +419,27 @@ Cassandra or Elasticsearch related command line options depending on which is us
 {{- include "elasticsearch.cmdArgs" . -}}
 {{- end -}}
 {{- end -}}
+
+{/*
+Return the appropriate apiVersion for ingress.
+*/}}
+{{- define "jaeger.ingress.apiVersion" -}}
```

Comment:
@mfuxi my bad for missing this, but can you remove these changes in helpers.tpl file completely.
And for check in side the ingress.yaml files, just use `"common.ingress.supportsIngressClassname"` to set the ingressClassName field.

Reply:
OK, so to sum things up:
change this in query,collector and hotrod ingress files:
`{{- $ingressSupportsIngressClassName := eq (include "jaeger.ingress.supportsIngressClassName" .) "true" }}`
Into this:
`{{- $ingressSupportsIngressClassName := eq (include "common.ingress.supportsIngressClassname" .) "true" }}`

And remove completely the additions I made in helpers.tpl

Reply:
OK, removed my addition from _helpers.tpl and changed the variable to the common chart variable

Reply:
OK, removed my addition from _helpers.tpl and changed the variable to the common chart variable

Reply:
there are still changes in helpers.tpl file which need to go. Since we don't nee them anymore.